src/genie/libs/parser/iosxr/show_bgp.py

```diff
@@ -9165,8 +9170,8 @@ def cli(self, address_family=None, output=None):
         # *>i192.168.111.0/24       10.189.99.98                                                    0       0 i
         # *> 10.7.7.7/32        10.10.10.107             0             0 65107.65107 ?
         p11 = re.compile(r'^(?P<status_codes>(i|s|x|S|d|h|\*|\>|\s)+)'
```

Comment:
Please comment above this what you're now matching

Reply:
same here

Reply:
@Kani999, kindly do not resolve the conversion unless you address the comments. You have to add the example for which you have made the changes in regex pattern. Kindly do so and then only resolve the conversion.

Reply:
Ok, Difference between Old and New regex:
-    Old one: https://regex101.com/r/w6F0eT/1
-    New One: https://regex101.com/r/18lHd3/1